src/graphene-ray.c

```diff
@@ -53,8 +53,11 @@
 #include <float.h>
 
 #if defined(_M_X64) && ! defined(isnanf)
-# define isnanf(x) _isnanf(x)
-# define HAVE_ISNANF
+# define my_isnan(x) _isnanf(x)
```

Comment:
Would it be possible to hoist this whole thing into `graphene-private.h` and call it `graphene_isnan()` instead? Something like:

```c
#if defined(_M_X64) && !defined(isnanf)
# define graphene_isnan(x) _isnanf(x)
#elif defined(HAVE_ISNANF)
# define graphene_isnan(x) isnanf(x)
#else
# define graphene_isnan(x) isnan(x)
#endif
```

Reply:
Definitely, I did it in the file directly because it was the only place using `isnan`.

Will let you know when I'm done.

Reply:
Done, thanks!